README.md

```diff
@@ -123,7 +123,7 @@ before they can be used. Each `Cabinet` subclass can expose a `set_configuration
 class method to take care of any required initial setup.
 
 ```python
-from cabinets.cabinet.s3 import S3Cabinet
+from cabinets.cabinet.s3_cabinet import S3Cabinet
```

Comment:
This is imported in a number of places in depending services so it will break, so we should upgrade to `0.5.0` :)